doc-content/enterprise-only/openshift/configmaps-con.adoc

```diff
@@ -0,0 +1,96 @@
+[id='configmaps-con_{context}']
+= KIE configuration and ConfigMaps
+
+When installing {PRODUCT} operator, the operator creates ConfigMaps, a YAML file, prefixed as `kieconfig-$VERSION`, for the current namespace. A ConfigMap is a YAML file containing the `DeploymentConfigs` for the {PRODUCT} components for that namespace.
```

Comment:
It is not only DeploymentCOnfigs, there is also secrets, pvc, routes, services, etc.

Reply:
Rephrased as "A ConfigMap is a YAML file containing the configuration for functionality such as `DeploymentConfigs`, secrets, routes, and services for the {PRODUCT} components for that namespace."